build.gradle

```diff
@@ -20,6 +20,8 @@ repositories {
 dependencies {
     compile 'org.apache.commons:commons-lang3:3.4'
     compile 'joda-time:joda-time:2.9.3'
+    compile 'au.com.bytecode:opencsv:2.4'
+    compile 'com.imsweb:seerapi-client-java:3.0'
```

Comment:
I thought you used SEER API in a lab class to create some CSV file, do you really this as "compile" instead of "testCompile"?